.github/pull_request_assignment.yml

```diff
@@ -180,7 +184,6 @@
       - lirenhe
       - weidongxu-microsoft
       - ArcturusZhang
-      - raych1
```

Comment:
Are you no longer going to be reviewing PRs?

Reply:
I'm assigned for a few rules above this rule~